### DISCUSSION TITLE
Looking for an exposed REST API for editing media metadata

### DISCUSSION THREAD

**Discussion by viv1** [mediacms-io/mediacms#673] [Feature Requests]
**Describe the feature you'd like**

I can see that "api/v1/media" is exposed to allow creating/posting new videos, and works well for my case. However, I don't see any way to modify the media metadata (tags, allow_download, is_reviewed, etc) through REST API other than through Django forms (which requires cookie & CSRF checks). Am I missing something or is this a functionality I need to add myself for now ?

Thank you.

**Comment by mgogoulos** [comment]
Hi, you are right, this is not implemented on the API, you'll have to extend it to support that functionality

**Reply by viv1** [reply]
Thanks for the quick response @mgogoulos . 
Btw, great job on mediacms.

**Reply by LaserKaspar** [reply]
It seems like this is still not possible. Is it possible to change default for the whole site so I don't have to "unlist"/"private" most of the videos I upload automatically to my instance? I.e. the videos are private by default?

EDIT: For anyone looking for the same thing: 
https://github.com/mediacms-io/mediacms/blob/ab4d9d67df0b2f4453012a397392650fa067400e/cms/settings.py#L22-L23
Can be changed for your instance (if using docker) in: `./deploy/docker/local_settings.py`
